utils/contentStyle.json

```
@@ -76,6 +76,14 @@
 						"correction": {
 							"replaceWith": "\\1\\2(\\3\\4)\\1"
 						}
+					},
+					{
+						"description": "accidental commonly doubled word",
+						"regex": "\\b(than|the|with|to)\\s+\\1\\b",
```

```suggestion
						"regex": "\\b(than|the|with|to|is|a|an|that|of|as)\\s+\\1\\b",
```

You can always add more... but this seems like a good idea.

---

"that that" is grammatically correct